common/bcp47/timezone.xml

```diff
@@ -280,6 +280,7 @@ For terms of use, see http://www.unicode.org/copyright.html
             <type name="mwblz" description="Blantyre, Malawi" alias="Africa/Blantyre"/>
             <type name="mxchi" description="Chihuahua, Mexico" alias="America/Chihuahua"/>
             <type name="mxcun" description="Cancún, Mexico" alias="America/Cancun"/>
+            <type name="mxcjs" description="Ciudad Juárez, Mexico" alias="America/Ciudad_Juarez"/>
```

Comment:
```suggestion
            <type name="mxcjz" description="Ciudad Juárez, Mexico" alias="America/Ciudad_Juarez"/>
```
There is no 's' in the name of the city.

Reply:
@markusicu We use LOCODE by default - https://service.unece.org/trade/locode/mx.htm

MX  CJS | Ciudad Juárez | Ciudad Juarez | CHH | ---4---- | AI | 0212 |   | 3144N 10629W
-- | -- | -- | -- | -- | -- | -- | -- | --

So "mxcjs" is correct.

Reply:
ok tnx never mind... but then the LOCODE is wrong :-P